Source/Core/Common/FileUtil.cpp

```diff
@@ -755,6 +765,8 @@ std::string GetSysDirectory()
 	sysDir = GetBundleDirectory() + DIR_SEP + SYSDATA_DIR;
 #elif defined (_WIN32)
 	sysDir = GetExeDirectory() + DIR_SEP + SYSDATA_DIR;
+#elif defined (LINUX_LOCAL_DEV)
```

Comment:
I'd probably do the same as in CommonPaths.h, and reuse the same block for both defines.